src/python/pants/fs/fs_test.py

```diff
@@ -81,20 +75,19 @@ def test_workspace_materialize_directories_result(self):
 
     digest, = self.scheduler.product_request(Digest, [input_files_content])
 
-    with temporary_dir() as tmp_dir:
-      path1 = Path(tmp_dir, 'a.txt')
-      path2 = Path(tmp_dir, 'subdir', 'b.txt')
+    path1 = Path('a.txt')
+    path2 = Path('subdir/b.txt')
 
-      self.assertFalse(path1.is_file())
-      self.assertFalse(path2.is_file())
+    assert not path1.is_file()
+    assert not path2.is_file()
```

Comment:
Is there any reason to prefer the assert keyword to calling `self.assert*` methods?

Reply:
Yes. Pytest has magic around `assert` to make it far more useful output. Specifically, Pytest will show how each value was derived, like this: https://docs.pytest.org/en/latest/assert.html#asserting-with-the-assert-statement

You can still use `self.assertFalse`—that's totally valid. Only this Pytest style brings additional benefits (including not having to remember what all the different `self.assertX` functions are called).